libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -98,13 +98,13 @@ bool NameAndTypeResolver::performImports(SourceUnit& _sourceUnit, map<string, So
 					{
 						m_errorReporter.declarationError(
 							imp->location(),
-							"Declaration \"" +
-							alias.symbol->name() +
-							"\" not found in \"" +
-							path +
-							"\" (referenced as \"" +
-							imp->path() +
-							"\")."
+							"Declaration" +
+							quote(alias.symbol->name()) +
```

Comment:
I wouldn't have guessed that adding spaces around would be the default implementation.

Reply:
I tried to use the most common case as default. Mabye using a different functionname instead is better? `quoteSp` for example?

Reply:
What is the advantage of implicit spaces?

Reply:
I felt `"This is an" + quote("example") + "message."` reads nicer than
`"This is an " + quote("example") + " message."`. Basically you need to almost always add those spaces extra and I tried to save the extra grunt work.

Reply:
Especially with multiple quotes in one sentence:
`"Elements" + quote("a") + "and" + quote("b") + "cannot be used in this context."`

Reply:
it's not a terrible issue and if you prefer I can just go with a normal `quote()` that does only quote and nothing else